core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -538,6 +538,10 @@ def __init__(self):
         # text into a feature a vector.
         self.count_vector = None
 
+    @property
+    def version(self):
+        return 1
```

Comment:
Should 1 be a global constant?

Reply:
I don't think so. This version will change with new iteration of classifiers.

Reply:
We can call it `algorithm_version`, schema is something that evolves over time and is backward compatible, however, classifiers won't be backward compatible and need to be recreated. So we can use `algorithm_version` to refer to this. And yes, this is expected to change in the future with new versions of classifiers.

Reply:
Changed to `algorithm_version`.

Reply:
Thanks. Maybe you can add a docstring explaining what it is and that it is expected to change in a non-backwards-compatible way (I know you have this in the parent class but it helps to see this or the relevant subset of it in the subclasses too).

Reply:
Removed the CodeClassifier files.